install.sh

```diff
@@ -134,24 +147,30 @@ fi
 echo_stage "Creating the config files of WSL-Hello-sudo..."
 set -x
 sudo mkdir -p /etc/pam_wsl_hello/
+PAM_CONFIG_FILENAME="/etc/pam_wsl_hello/config"
+AUTHENTICATOR_PATH="authenticator_path = \"$PAM_WSL_HELLO_WINPATH/WindowsHelloBridge.exe\""
+
 set +x
-if [ ! -e "/etc/pam_wsl_hello/config" ] || prompt_yn "'/etc/pam_wsl_hello/config' already exists. Overwrite it? [y/N]" "n" ; then
+if [ ! -e "$PAM_CONFIG_FILENAME" ] || prompt_yn "'$PAM_CONFIG_FILENAME' already exists. Overwrite it? [y/N]" "n" ; then
   set -x
   sudo touch /etc/pam_wsl_hello/config
-  sudo echo "authenticator_path = \"$PAM_WSL_HELLO_WINPATH/WindowsHelloAuthenticator/WindowsHelloAuthenticator.exe\"" | sudo tee /etc/pam_wsl_hello/config
+  sudo echo "$AUTHENTICATOR_PATH" | sudo tee "$PAM_CONFIG_FILENAME"
   sudo echo "win_mnt = \"$MNT\"" | sudo tee -a /etc/pam_wsl_hello/config
+elif grep -q "WindowsHelloAuthenticator" "$PAM_CONFIG_FILENAME" ; then
+  echo "Migrating PAM config to v2.0"
+  sed -i "1s/.*/$AUTHENTICATOR_PATH/" "$PAM_CONFIG_FILENAME"
```

Comment:
This causes the following error because `$AUTHENTICATOR_PATH` contains `/`. Let's update the sed command.

```console
sed: -e expression #1, char 31: unknown option to `s'
```

```suggestion
  sed -i "1s;.*;$AUTHENTICATOR_PATH;" "$PAM_CONFIG_FILENAME"
```